components/tcf/ui/src/TCFContainer/TCFContainer.js

```diff
@@ -0,0 +1,66 @@
+import React, {Suspense, useState, useEffect} from 'react'
+import PropTypes from 'prop-types'
+
+const CONSENT_STATUS_NOT_ACCEPTED = 'NOT_ACCEPTED'
+const FirstLayer = React.lazy(() => import('@s-ui/sui-tcf-first-layer'))
+const SecondLayer = React.lazy(() => import('@s-ui/sui-tcf-second-layer'))
+
+export default function TCFContainer({
+  getVendorList,
+  getConsentStatus,
+  loadUserConsent,
+  saveUserConsent
+}) {
+  const [showFirstLayer, setShowFirstLayer] = useState(false)
+  const [showSecondLayer, setShowSecondLayer] = useState(false)
```

Comment:
You can unify all in one state and avoid multiple possible rerenders in lines 29 and 30.
Maybe a showLayer that can be false(no show), 1(for firstlayer) or 2(for secondLayer)